spec/lib/ability_spec.rb

```diff
@@ -104,10 +104,11 @@
     end
   end
 
-  context "a coalition lead" do
-    let(:coalition_member_organization) { create(:vita_partner) }
+  xcontext "a coalition lead" do
```

Comment:
Why skip this?

Reply:
because i didn't want to go and make the ability file work for all the new roles (this test applied to our previous "implementation" of coalition leads) - i guess i could also just delete the test instead - thoughts?

Reply:
Seems peaceful to leave a test with some hints for what coalition lead will look like!